### PR TITLE
CDAP-14815 SQL Server plugin

### PR DESCRIPTION
**Pull request by ibarbashov**
Sql Server plugin feature doesn't have oracle-plugin in history, after review I'll rebase it

### CODE REVIEW

File: mssql-plugin/src/main/java/co/cask/mssql/SqlServerPostAction.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    @Name(SqlServerConstants.INSTANCE_NAME)
+    @Description("The SQL Server instance name to connect to. When it is not specified, a connection is made" +
+      " to the default instance. For the case where both the instanceName " +
+      "and port are specified, see the notes for port.")
+    @Nullable
+    public String instanceName;
+
+    @Name(SqlServerConstants.QUERY_TIMEOUT)
+    @Description("he number of seconds to wait before a timeout has occurred on a query. The default value is -1, " +
```
Comment (albertshau) [inline mssql-plugin/src/main/java/co/cask/mssql/SqlServerPostAction.java]: typo: he -> The

Comment (albertshau) [reply mssql-plugin/src/main/java/co/cask/mssql/SqlServerPostAction.java]: other plugins have this typo as well

Comment (ibarbashov) [reply mssql-plugin/src/main/java/co/cask/mssql/SqlServerPostAction.java]: done

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**